src/components/Calculator.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+class Calc extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
```

Comment:
- [ ] Nice way of declaring the calc component but it is not what required, this is a class component while you are asked to create the calculator as a functional component. please make the changes for your code too like: ---> 

` function Calculator() {
  return <div> ..... </div>
}
`

Reply:
Alright. Thank you for highlighting that